src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -176,6 +195,152 @@ class WalletService(
             }
     }
 
+    fun validateWallet(
```

Comment:
maybe you can rename the method name, a session of a wallet is validated and not the wallet itself

Reply:
changed to `validateWalletSession`